README.md

```diff
@@ -106,8 +110,62 @@ var lorem = "ipsum \"dolor\" sit 'amet'";
 var maecennas = "ipsum 'dolor' sit \"amet\"";
 ```
 
+## Examples for object key normalization
+
+Given this input program:
+
+```js
+var famousQuoteBy = {
+  'Princess Leia': 'I love you!',
+  'Han "Shot first" Solo': 'I know!',
+  Spock: 'Fascinating.',
+  "Dr. Emmett 'Doc' Brown": '1.21 gigawatts???'
+};
+
+```
+
+### {type: 'single', normalizeObjectKeys: true}
+```js
+var famousQuoteBy = {
+  'Princess Leia': 'I love you!',
+  'Han "Shot first" Solo': 'I know!',
+  'Spock': 'Fascinating.',
```

Comment:
I don't think we necessarily want this behavior (i.e. quoting unquoted keys). There are coding styles where users only want quotes when the key is a keyword (e.g. `delete`, `var`).

```js
{
  foo: 'bar',
  'delete': false
}
```